test/common.py

```diff
@@ -1532,7 +1532,7 @@ def run_browser(self, html_file, expected=None, message=None, timeout=None, extr
             if extra_tries > 0:
               print('[test error (see below), automatically retrying]')
               print(e)
-              return self.run_browser(html_file, message, expected, timeout, extra_tries - 1)
+              return self.run_browser(html_file, expected, message, timeout, extra_tries - 1)
```

Comment:
Is this related to this PR?

Reply:
No, just a drive-by fix for an issue I noticed while running browser tests.